packages/salesforcedx-apex-replay-debugger/test/unit/adapter/apexReplayDebugVariablesHandling.test.ts

```diff
@@ -636,6 +637,71 @@ describe('Replay debugger adapter variable handling - unit', () => {
         ) as ApexVariableContainer;
         expect(updatedNonRefVariable.value).to.be.eq('5');
       });
+
+      it('Should correctly deal with circular references and variable values', () => {
```

Comment:
Please clean up needless comments.

Reply:
Nothing in that test is a 'needless' comment. Please be specific about which ones you think are needless.